cmd/ops/deprecation.go

```diff
@@ -29,7 +31,14 @@ var (
 func init() {
 	Program.AddCommand(deprecationsCmd)
 	deprecationsCmd.Flags().StringVar(&schemaFile, schemaFileFlag, "", "Server's schema file (required)")
+	deprecationsCmd.Flags().BoolVar(&xcodeFormat, xcodeFormatFlag, false, "Override output format to xcode [EXPERIMENTAL]")
 	deprecationsCmd.MarkFlagRequired(schemaFileFlag) //nolint:errcheck // will err if flag doesn't exist
+
+	// TODO: This is required because the test suite doesn't finish the program and flags are not reset. Find a better way to do this.
```

Comment:
We can run the test off process to avoid this, but that means building the program. Other solution would be to test the commands passing the flags and args as parameters instead of with cmdtest

Reply:
I think this is OK for now, without the `TODO`